.travis.yml

```diff
@@ -9,7 +9,7 @@ notifications:
 env:
   global:
     - ADDITIONAL_DEBS='apt-utils dialog curl wget'
-    - AFTER_SCRIPT='rosenv sh .travis.xacro_test.sh'
+    - AFTER_SCRIPT='rosenv bash .travis.xacro_test.sh'
```

Comment:
@ipa-mdl 
this PR would break the test in `industrial_ci`...do you have any comment?

Reply:
I wondered why you chose `sh` in the first place..